Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -179,6 +180,7 @@ public override void Update()
 			else
 			{
 				if ( player.controls.holdButton &&
+				     player.stats.CanUseStat( Stat.Stamina ) &&

```

Comment:
Yup, I just wasn't checking if the player had enough stamina before trying to climb.